### ISSUE TITLE
Advanced macros

### ISSUE THREAD

**Issue by marcos-diaz** [inputlabs/organization#68]
Advanced macros, removing the limitations of current implementation:

- Fixed timing.
- No modifier keys (control, shift, alt).
- Not able to run multiple macros in parallel.

**Comment by UltraBlackLinux** [comment]
Already mentioned it, but I'm gonna leave this here: https://github.com/Ryochan7/sc-controller

It would allow for all of this advanced mapping stuff. SCC is extremely flexible in what you can do.

If support for that tool was implemented/the tool supported alpakka it would be pretty easy to create advanced mappings.

The only issue I'm currently seeing is that SDL controller mappings have quite a few less buttons, but maybe Ryochan7 knows of a way to make this work regardless

**Comment by marcos-diaz** [comment]
Tracked in FW repo issues
